qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -426,7 +426,7 @@ DEV=
 
 # test detach/attach with --snap-id
 SNAPID=`rbd snap ls ${POOL}/${IMAGE} | awk '$2 == "snap" {print $1}'`
-OUT=`_sudo rbd device --device-type nbd --options try-netlink,show-cookie map --snap-id ${SNAPID} ${POOL}/${IMAGE}`
+OUT=`_sudo rbd device --device-type nbd --options show-cookie map --snap-id ${SNAPID} ${POOL}/${IMAGE}`
```

Comment:
Stick to the shorter syntax:
```suggestion
OUT=`_sudo rbd device --device-type nbd --show-cookie map --snap-id ${SNAPID} ${POOL}/${IMAGE}`
```